src/main/resources/swagger/rawls.yaml

```diff
@@ -789,10 +792,10 @@ paths:
           schema:
             $ref: '#/definitions/ErrorReport'
         '502':
-          description: Trouble talking to Cromwell
+          description: Trouble talking to Agora
```

Comment:
I believe this is too specific - we could get a 502 from Agora here trying to pull the WDL down, or a 502 from Cromwell trying to submit, or a 500 from either of those, etc - basically I think we just bubble up errors from either - not sure if we document every case or leave this out of docs?